mvpa2/tests/test_hyperalignment.py

```diff
@@ -219,8 +219,8 @@ def test_hpal_svd_combo(self):
                            k=0)
             sv_corrs.append(ndcs)
         self.assertTrue(
-            np.all(np.abs(np.array(sv_corrs)) >= 0.99),
-            msg="Hyperalignment with dimensionality reduction should have"
+            np.all(np.abs(np.array(sv_corrs)) >= 0.9),
```

Comment:
Yes, for some random test datasets when added extra "bogus" features correlate with original principal dimensions.